src/app/ipv/middleware.test.js

```diff
@@ -1298,7 +1298,7 @@ describe("journey middleware", () => {
     });
 
     it("should set journey to UPDATE_NAMES_DOB if detailsToUpdate is givenNames and lastName", async function () {
```

Comment:
We should probably update these test descriptions to match the changes made, or make them more generic